lib/car.rb

```diff
@@ -0,0 +1,36 @@
+class Car
+  attr_reader :make, :model, :payments, :loan, :color
+
+  def initialize(type, monthly_payment, loan_length)
+    @make = type.split.first
+    @payments = monthly_payment
+    @loan = loan_length
```

Comment:
I would have used `loan_length` rather than `loan` and `monthly_payment` rather than `payments` as the instance variable as well. That was you wouldn't have to create the methods (monthly_payment line 14  or loan_length line 18) below.
```
attr_reader :loan_length, :monthly_payment

def initialize(type, monthly_payment, loan_length)
  ...
  @loan_length = loan_length
  @monthly_payment = monthy_payment
end
```